extension/src/DocumentFunctions.ts

```diff
@@ -87,6 +76,25 @@ export async function openAlFileFromXliffTokens(
   return openTextFileWithSelectionOnLineNo(obj.objectFileName, codeLineIndex);
 }
 
+export function getObjectFromTokens(
+  alObjects: ALObject[],
+  tokens: XliffIdToken[]
+): ALObject {
+  const obj = alObjects.find(
+    (x) =>
+      x.objectType.toLowerCase() === tokens[0].type.toLowerCase() &&
+      x.objectName.toLowerCase() === tokens[0].name.toLowerCase()
+  );
```

Comment:
Why pass `tokens` as an array?

Reply:
I just used "extract method", it was like that before 😁

Reply:
Change or leave as is?

Reply:
I'm using both the first token and then all of them in the error message, so I think we'll leave it there